plugin/pkg/scheduler/algorithm/predicates/error.go

```diff
@@ -40,8 +40,8 @@ var (
 	ErrNodeLabelPresenceViolated = newPredicateFailureError("CheckNodeLabelPresence")
 	ErrServiceAffinityViolated   = newPredicateFailureError("CheckServiceAffinity")
 	ErrMaxVolumeCountExceeded    = newPredicateFailureError("MaxVolumeCount")
-	ErrNodeUnderMemoryPressure   = newPredicateFailureError("NodeUnderMemoryPressure")
-	ErrNodeUnderDiskPressure     = newPredicateFailureError("NodeUnderDiskPressure")
+	ErrNodeUnderMemoryPressure   = newPredicateFailureError("NodeNotUnderMemoryPressure")
```

Comment:
Actually, I think the error messages are correct. These errors are returned when a node "is" under memory pressure or disk pressure. I think we should change `NoNodeAvailableMsg` to say something like:

`No nodes are available that match all of the predicates. The following predicate failures are observed for a set of nodes:`

Reply:
In that particular case the more friendly error message would be `All nodes matched NodeUnderMemoryPressure predicate. No free nodes available."

Reply:
We should create an error message that works when various predicates fail for different set of nodes. For example, 5 nodes failed "NodeUnderMemoryPressure", 3 nodes failed "NodeUnderDiskPressure", 2 nodes failed "NodeNotReady".

Reply:
"0/10 nodes available - 5 NodeUnderMemoryPressure, 3 NodeUnderDistPressure, 2 NodeNotReady."

Reply:
Sorry, i'm late, thanks for your responses. I will think carefully for your suggestions.